### PR TITLE
Column properties

### PR DESCRIPTION
**Pull request by marygabry**
Added functionalities to use the ColumnAttribute, if defined, from net451 in Get, Insert, Update and Delete extensions methods.

### CODE REVIEW

File: Dapper.Contrib/SqlMapperExtensions.Async.cs [hunk 1]
```diff
@@ -24,18 +24,34 @@ public static partial class SqlMapperExtensions
         /// <returns>Entity of T</returns>
         public static async Task<T> GetAsync<T>(this IDbConnection connection, dynamic id, IDbTransaction transaction = null, int? commandTimeout = null) where T : class
         {
+            String keyName = "";
```
Comment (chriswolf-nrg) [inline Dapper.Contrib/SqlMapperExtensions.Async.cs]: You don't need this variable.

File: Dapper.Contrib/SqlMapperExtensions.Async.cs [hunk 2]
```diff
@@ -24,18 +24,34 @@ public static partial class SqlMapperExtensions
...
+
             if (!GetQueries.TryGetValue(type.TypeHandle, out string sql))
             {
                 var key = GetSingleKey<T>(nameof(GetAsync));
                 var name = GetTableName(type);
+                keyName = key.Name;
+                String keyColumnName = GetDBColumnName(key);
 
-                sql = $"SELECT * FROM {name} WHERE {key.Name} = @id";
+                sql = $"select * from {name} where {keyColumnName} = @" + keyName;
```
Comment (chriswolf-nrg) [inline Dapper.Contrib/SqlMapperExtensions.Async.cs]: You can just do this:
`sql = $"select * from {name} where {keyColumnName} = @{key.Name}";`

Comment (marygabry) [reply Dapper.Contrib/SqlMapperExtensions.Async.cs]: This has been received and checked in. Thank you!

**Comment by NickCraver** [comment]
I can't speak to Contrib's future plans here and if this is a good fit, but please see #1658 - Dapper.Contrib is not located at https://github.com/DapperLib/Dapper.Contrib (separate from this repository). 

Note: I based the Dapper.Contrib fork on the original codebase which hopefully makes porting pull requests against the main branch there much easier - the intent was to split, not throw away work the community wants to go with.

**Pull request closed without merging**